src/index.ts

```diff
@@ -617,173 +445,11 @@ function initialSetup() {
 			}
 		});
 
-		socket.on('device_listen', function(deviceId: string, listenerType: string) { // emitted by a socket (tally page) that has selected a Device to listen for state information

```

Comment:
This socket event is used in "Tally" page that now is not working.

Reply:
I'll update the Web Tally. It should also use the new connection api with the `listenerclient_connect` event.